### ISSUE TITLE
Disable package in CSS

### ISSUE THREAD

**Issue by davidalejandroaguilar** [BadStreff/BetterAutoIndent#1]
Would you know of a way to disable this while editing CSS? It shouldn't do this there, in my opinion.

**Comment by badstreff** [comment]
I agree, I mostly write python so I can see if causing issues in css. It might be possible to do this in the keybinding file based on some context, that's probably the easiest and wouldn't require any code modification.

If it turns out I can't do it that way there is probably a way to add a preference setting to exclude certain files.
